src/main/java/core/mate/academy/model/Bulldozer.java

```diff
@@ -1,9 +1,5 @@
 package core.mate.academy.model;
 
-/**
- * Add some custom fields that could be only in Bulldozer
- * Do not remove no-args constructor
- */
 public class Bulldozer extends Machine {
     public Bulldozer() {
     }
```

Comment:
Machine contains fields general to  all subclasses so I would include constructor with parameters and pass them using super();  The same in all classes that extends machines